packages/toolpad-app/src/toolpad/ToolpadShell.tsx

```diff
@@ -107,7 +113,11 @@ function UpdateBanner() {
   return (
     <React.Fragment>
       {latestRelease && process.env.TOOLPAD_TARGET === 'CE' ? (
```

Comment:
Instead of adding it here, perhaps we can use
```tsx
enabled: process.env.TOOLPAD_TARGET !== 'CLOUD'
```
on the `useQuery` so that the query doesn't have to run at all on cloud?